app/(dashboard)/admin/service-catalog/merge-suggestions/page.tsx

```
@@ -0,0 +1,21 @@
+    headers: await headers()
+  });
+
+  if (!session) {
+    return redirect('/login')
+  }
+
+  if (session.user.role !== "admin") {
+    return redirect('/dashboard')
+  }
+
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)